README.md

```diff
@@ -13,21 +13,32 @@ Darkflame Universe is licensed under AGPLv3, please read [LICENSE](LICENSE). Som
 * You must disclose any changes you make to the code when you distribute it
 * Hosting a server for others counts as distribution
 
-## Disclaimers
```

Comment:
Nit: this moves the other level 3 headings into the previous section

Reply:
idk how to do markdown

Reply:
if you know how to do it better, please make th necessary markdown edits to the branch as I do not know best practices for this format

Reply:
```suggestion
## Disclaimers
```